provider/backup_utils.py

```diff
@@ -315,11 +317,18 @@ def blockdev_batch_backup(vm, source_lst, target_lst,
                 bitmap_data["persistent"] = persistent
             actions.append({"type": bitmap_add_cmd, "data": bitmap_data})
 
+        if disabled_bitmap_lst:
+            bitmap_data = {"node": source_lst[idx], "name": disabled_bitmap_lst[idx]}
+            actions.append({"type": bitmap_disable_cmd, "data": bitmap_data})
+
     arguments = {"actions": actions}
     if completion_mode == 'grouped':
         arguments['properties'] = {"completion-mode": "grouped"}
     vm.monitor.cmd("transaction", arguments)
-    list(map(lambda x: job_utils.wait_until_block_job_completed(vm, x, timeout), jobs_id))
+    if wait:
+        list(map(
+            lambda x: job_utils.wait_until_block_job_completed(vm, x, timeout),
+            jobs_id))
```

Comment:
Suggest adding a commit for this file.
Because if someone who git log to check this file will show commit message `Live backup via pull mode`, it is not clear.

Reply:
split into separate commits for provider